src/components/molecules/HomeTextBox.vue

```diff
@@ -1,19 +1,19 @@
 <template>
 	<div class="home__text-box">
-		<homeTextWHite v-bind:textWhite="text.textWhite"/>
-		<homeTextWHite v-bind:textCrimson="text.textCrimson"/>
+		<HomeTextWhite v-bind:textWhite="text.whiteText"/>
```

Comment:
why cant you create a text component which you can specify color?
`<text>` with default styling
`<text :color="white">` for light color
so you can get a smaller amount of components, so if you will have 20 components on page and every component will have its own atom for text it will be 20+ repeating components, instead of that you can create reusable component
example: https://bootstrap-vue.org/docs/components/button - its 1 button component that have a **lot** of variants that can be passed as a prop